### ISSUE TITLE
Move #bco-wc-form inside the checkout form

### ISSUE THREAD

**Issue by DanielMarklund** [Billmate/billmate-checkout-for-woocommerce#133]
We have a custom country switcher on our checkout page. It's a select element with the available billing countries. The default selection is the current customer's billing country.

**Expected behaviour (works in the old BCO):**
When selecting another country the original #billing_country field gets updated and triggers an update_checkout JS event. When this trigger happens, I check which country is selected and disable the BCO if Sweden (SE) isn't selected.

**Current behaviour:** 
When selecting another country the original #billing_country field gets updated. No event is triggered. In the current BCO the original billing fields are in a hidden #bco-wc-form element. To make the update_checkout JS event trigger, the #bco-wc-form needs to be inside of the checkout form element ( $('form[name=checkout]') ).

**Example Markup & JS that can be placed anywhere on the checkout page:**
```
<div class="billing-country-switcher">
    <?php woocommerce_form_field( 'billing_country_standalone', array( 'type' => 'country', 'default' => WC()->customer->get_billing_country() ) ); ?>
</div>

  <script type="text/javascript">
    jQuery('#billing_country_standalone').on('change', function () {
      var countrycode = jQuery(this).select2().val()
      jQuery("#billing_country").select2().val(countrycode).trigger('change')
    })
  </script>
```

**Comment by DanielMarklund** [comment]
This is a temporary fix we'll use, which moves add_wc_form to the order review:

```
$BCO_templates = new BCO_Templates;
remove_action( 'bco_wc_after_wrapper', 'add_wc_form', 10 );
add_action( 'bco_wc_before_order_review', array($BCO_templates, 'add_wc_form'), 10 );
```

**Comment by DanielMarklund** [comment]
Adding the code above also fixes #132

**Comment by NiklasHogefjord** [comment]
Hi Daniel!

Thank you for your suggestion and sorry for the late reply.

I have done some changes to the template file due to another fix. I agree with your proposal and during my work I have moved #bco-wc-form so it now is inside the Woo checkout form tag. The change can be found and tested in this branch: https://github.com/Billmate/billmate-checkout-for-woocommerce/tree/issue-checkout-layout.

I hope we can release this in the beginning of next week.
